cogboard-webapp/src/components/Widget.js

```diff
@@ -26,6 +26,15 @@ const StyledCard = styled(({
   box-shadow: none;
   grid-column-start: ${({ goNewLine }) => goNewLine === true && 1};
   grid-column-end: span ${({ columns }) => columns};
+  height: 100%;
```

Comment:
Is this really has to be that complicated to move this button down ?

Reply:
It is a little bit less complicated :)